src/overlays/actors/ovl_En_Mt_tag/z_en_mt_tag.c

```diff
@@ -153,25 +153,27 @@ s32 EnMttag_GetCurrentCheckpoint(Actor* actor, PlayState* play, s32* upcomingChe
     checkpointIterator = sStartingCheckpointPerSceneExitIndex[sceneExitIndex];
 
     // Iterates through all possible checkpoints that are associated with this sceneExitIndex.
+    //! @bug - sStartingCheckpointPerSceneExitIndex is indexed out of bounds when sceneExitIndex is 18, due to the
```

Comment:
Hmm, I'm wondering if the bug tag actually makes more sense with the bounds checking above:
`if ((sceneExitIndex < 4) || (sceneExitIndex >= 19))`

Reply:
I feel like you can argue that it makes sense for the sStartingCheckpointPerSceneExitIndex too (and claim that there should be an extra value of e.g. 46 at the end) so I personally prefer putting it by where the 'illegal' access takes place. But can make this change if you'd prefer

Reply:
I think I prefer with the bounds check, as the whole algorithm probably shouldn't even be running on the final point (kind of curious what happens if you do update the bounds check to `>= 18` but you don't have to check). However it does seem fine where it is anyways.

Reply:
Bump

Reply:
Actually, I see @hensldm approved this. What's the state of this convo?

Reply:
My preference is to move the bug tag up, but am fine with leaving it where it is so approved it.

Reply:
Moved it up now 👍🏼 let me know if the wording should change at all